bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/core/IBulkMutation.java

```diff
@@ -18,15 +18,20 @@
 import com.google.api.core.ApiFuture;
 import com.google.api.core.InternalApi;
 import com.google.cloud.bigtable.data.v2.models.RowMutationEntry;
+import com.google.cloud.bigtable.grpc.BigtableTableName;
 import java.io.IOException;
 
 /**
  * Interface to support batching multiple {@link RowMutationEntry} request into a single grpc *
  * request.
  *
  * <p>For internal use only - public for technical reasons.
+ *
+ * <p>See {@link
+ * com.google.cloud.bigtable.grpc.BigtableSession#createBulkMutation(BigtableTableName)} as a public
+ * alternative.
```

Comment:
For consistency, would you mind adding similar for:

* https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataClientWrapper.java
* https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutationWrapper.java
* https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutationWrapper.java
* https://github.com/googleapis/java-bigtable-hbase/blob/master/bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminClientWrapper.java

As well as the methods:
* BigtableSession#getDataClientWrapper
* BigtableSession#createBulkMutationWrapper
* BigtableSession#getTableAdminClientWrapper

Reply:
yes sorry, this is still WIP!